s2/cellunion.go

```diff
@@ -553,7 +553,7 @@ func (cu *CellUnion) encode(e *encoder) {
 	}
 }
 
-// Decode decodes the CellUnion.
+// Decode decodes the CellUnion. The resulting CellUnion is not check for validity.
```

Comment:
Typo: "is not checked"

Probably clearer to explicitly mention`IsValid` though.